### DISCUSSION TITLE
pre-compiled versions of AdminerEvo for specific drivers/languages

### DISCUSSION THREAD

**Discussion by devinemke** [adminerevo/adminerevo#84] [Ideas]
Is it possible to have pre-compiled versions of AdminerEvo for specific drivers/languages, much like what is available at https://www.adminer.org ? For example I only use Adminer for MySQL in English, and there is a ready made compile in the **_Downloads_** section at https://www.adminer.org. Perhaps these could be automatically made when a new stable version is released, and then either hosted here or at https://docs.adminerevo.org. Possible?

**Comment by LionelLaffineur** [comment]
Hi @devinemke, I find this a good idea.  Let me look what I can do !

**Comment by LionelLaffineur** [comment]
Ready at https://download.adminerevo.org !
